build-tools/p2p-v2-test/docker-compose.yml

```diff
@@ -0,0 +1,199 @@
+version: "3.8"
+
+services:
+  node01:
+    build:
+      context: ../..
+      dockerfile: build-tools/p2p-v2-test/Dockerfile
+    container_name: $BASE_NAME-node01
+
+    volumes:
+      - data01:$DATA_DIR_IN_CONTAINER
```

Comment:
Instead of using these volumes and then going through hell to find the data directory, isn't it easier to make data01 a local data directory in the same directory of docker-compose, so with something like this:

```
    volumes:
      - data/node-data01:$DATA_DIR_IN_CONTAINER
```

This way, the data is accessible locally and the issue you mentioned in the readme file about the cookie wouldn't be really there. One only has to specify the port and data directory.

Reply:
You mean, create the directory inside the source tree? Well, I personally wouldn't like that. IMO it should be a separate directory, outside of the source tree.

Also, the custom directory would still be owned by root, so it's not a huge improvement.
It's possible to make the current user own it, I tried to do that initially. But it's extra hassle and AFAIR you have to pre-create the mount points before running `docker compose`, so that they're owned by the current user, which means you have to wrap `docker compose` in a bash script or something.